docs/extend/plugins_volume.md

```diff
@@ -12,10 +12,10 @@ parent = "mn_extend"
 
 Docker volume plugins enable Docker deployments to be integrated with external
 storage systems, such as Amazon EBS, and enable data volumes to persist beyond
-the lifetime of a single Docker host. See the [plugin documentation](plugins.md)
+the lifetime of a single Docker host. See the [plugin documentation](plugin_api.md)
```

Comment:
the plugins page refers people to plugin_api.md, so probably better to link to that directly